test/e2e/charts/flow-visibility/templates/clickhouseinstallation.yaml

```diff
@@ -0,0 +1,94 @@
+{{- define "clickhouse.tlsConfig" -}}
+{{- $Files := .Files }}
+{{- $Global := .Global }}
+{{- range $path, $_ :=  .Files.Glob  "provisioning/tls/*" }}
+{{ regexReplaceAll "(.*)/" $path "" }}: |
+{{ tpl ($.Files.Get $path) $Global | indent 2 }}
+{{- end }}
+{{- end -}}
+
+apiVersion: "clickhouse.altinity.com/v1"
+kind: "ClickHouseInstallation"
+metadata:
+  name: clickhouse
+  labels:
+    app: clickhouse
+  namespace: flow-visibility
+spec:
+  configuration:
+    users:
+      clickhouse_operator/k8s_secret_password: flow-visibility/clickhouse-secret/password
+      clickhouse_operator/networks/ip: "::/0"
+    profiles:
+      readonly/readonly: 1
+    {{- if .Values.secureConnection.enable }}
+    files:
+      {{- include "clickhouse.tlsConfig" (dict "Files" .Files "Global" .) | indent 6 }}
+    {{- end }}
+    clusters:
+      - name: "clickhouse"
+        {{- if .Values.secureConnection.enable }}
+        secure: "yes"
+        settings:
+          tcp_port: 9000
+          tcp_port_secure: 9440
+          https_port: 8443
+          http_port: 8123
+        {{- end }}
```

Comment:
I would expect `tcp_port_secure` and `https_port` to be gated by `.Values.secureConnection.enable`, but why is it also the case for `tcp_port` and `http_port`?

Reply:
It's a mistake. Only `secure`, `https_port: 8443` and `tcp_port_secure: 9440` should be gated. Fixed. Thanks.